### PR TITLE
added Player app

### PR DESCRIPTION
**Pull request by masihtamsoy**

### CODE REVIEW

**Comment by masihtamsoy** [comment]
Code review kr skate h yawn line wise

**Comment by tanmay1618** [comment]
Sahi

File: Player/admin.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+
+# Register your models here.
+from .models import Player
+
+admin.site.register(Player)
```
Comment (masihtamsoy) [inline Player/admin.py]: change line here... as code comparison is not possible.. anaconda me yahn pr wo indicate kr rh hota !

File: Player/models.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
```
Comment (masihtamsoy) [inline Player/models.py]: mongo dalla hi naahi mc

File: Player/urls.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^add/', views.addPlayer, name='AddPlayer'),
+    url(r'^edit/', views.editPlayer, name='EditPlayer'),
+    url(r'^delete/', views.deletePlayer, name='DeletePlayer'),
+    ]
```
Comment (masihtamsoy) [inline Player/urls.py]: yahn pr phi change line

File: Player/views.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import json
+# Create your views here.
+
+@csrf_exempt
+def addPlayer(request):
+	response_data ={}
+	if(request.method == 'POST'):
+		#pdb.set_trace()
+		PlayerName = request.POST['name']
+		PlayerEmailId = request.POST['email_id']
```
Comment (masihtamsoy) [inline Player/views.py]: player_name ... yahn pr variable naming convention underscore hota h.. python wall feel aayega

**Pull request merged**